src/test/e2e/create-plugin.test.ts

```diff
@@ -45,4 +45,24 @@ describe('Basic UI Tests', () => {
         const item = await section.findItem(randomName);
         assert.notStrictEqual(undefined, item);
     }).timeout(40000);
+
+    it('Should create a new DataPlugin from sample file', async () => {
+        const randomName: string = Guid.create().toString();
+        await new Promise(res => setTimeout(res, 5000)); // wait for loading vscode completely

```

Comment:
Doesn't provide vscode anything to avoid this timeouts? Timeouts will sooner or later cause flaky tests.